### PR TITLE
Storyboard fix and indexes

### PR DESCRIPTION
**Pull request by EmilySunpy**
Removed misinformation as floating point numbers are allowed in move commands.
Fixed order of which origin is mentioned as it did not match the indexes before.
Added numbers for indexes for layers and origin in lists.
(Useful for those who make storyboard creation tools)

---

### CODE REVIEW

**Comment by TPGPL** [comment]
could you give a source for those informations, please?

File: wiki/Storyboard_Scripting/Commands/en.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ See the [shorthand](/wiki/Storyboard_Scripting/Shorthand) section for an explana
 
 | Affects | Value definition | Default value |
 | ------- | ---------------- | ------------- |
-| The location of the object in the play area. | An (x,y) position, as specified above. Decimals are not allowed. | An (x,y) position, as specified above. Decimals are not allowed. |
+| The location of the object in the play area. | An (x,y) position, as specified above. Decimals are accepted. | An (x,y) position, as specified above. Decimals are accepted. |
```
Comment (cl8n) [inline wiki/Storyboard_Scripting/Commands/en.md]: for default value, it'd be the initial sprite/animation position

File: wiki/Storyboard_Scripting/Cheat_Sheet/en.md [hunk 2]
```diff
@@ -41,24 +41,25 @@ Z-order (back to front) is determined by the order the files appear in the .osu
...
--   TopLeft
--   TopCentre
--   TopRight
--   CentreLeft
--   Centre
--   CentreRight
--   BottomLeft
--   BottomCentre
--   BottomRight
+-   0 - TopLeft
```
Comment (cl8n) [inline wiki/Storyboard_Scripting/Cheat_Sheet/en.md]: I would just add a note here that 6 can also be TopLeft, but you shouldn't use it. Custom is not used for storyboarding

**Comment by cl8n** [comment]
@TPGPL it's correct, a lot of storyboarders use numbers instead of full names now because it saves filesize when you have tons of effects

**Comment by yugecin** [comment]
can confirm they're correct

you can see the values in the osu source, see https://github.com/ppy/osu/blob/5e5a74293e2710690328dda0086ec0fe9617ac24/osu.Game/Beatmaps/Formats/LegacyDecoder.cs#L150

File: wiki/Storyboard_Scripting/Commands/en.md (hunk 1 above)
Comment (yugecin) [reply wiki/Storyboard_Scripting/Commands/en.md]: see also #1660 I guess

File: wiki/Storyboard_Scripting/Cheat_Sheet/en.md (hunk 2 above)
Comment (cl8n) [reply wiki/Storyboard_Scripting/Cheat_Sheet/en.md]: doesn't have to be confusing, just note that it does the same thing as topleft. if a storyboarder did use it for some reason, other people might want to know what 6/Custom means

Comment (yugecin) [reply wiki/Storyboard_Scripting/Cheat_Sheet/en.md]: okay, readded them

**Comment by Wieku** [comment]
Also can confirm it's correct, without looking at lazer's code my storyboard parser in Go wouldn't work properly.

File: wiki/Storyboard_Scripting/Cheat_Sheet/en.md [hunk 3]
```diff
@@ -41,24 +41,25 @@ Z-order (back to front) is determined by the order the files appear in the .osu
...
+-   1 - Centre
+-   2 - CentreLeft
+-   3 - TopRight
+-   4 - BottomCentre
+-   5 - TopCentre
+-   6 - Custom (same effect as TopLeft but shouldn't be used)
```
Comment (TPGPL) [inline wiki/Storyboard_Scripting/Cheat_Sheet/en.md]: comma before but
shouldn't -> should not

Comment (EmilySunpy) [reply wiki/Storyboard_Scripting/Cheat_Sheet/en.md]: This was fixed over a month ago... maybe you should stop blocking? >.>

**Pull request merged**